scripts/busybox-ping-test/README.md

```diff
@@ -44,60 +47,105 @@ You can configure the alcor services name, port number in the following way:
 2. vpc_manager            = {"name":"vpm",    "port":9001}
 3. subnet_manager         = {"name":"snm",    "port":9002}
 
-With the above configuration the ignite service will be run as name 'ignite',
+With the above configuration the ignite service will be run with the name 'ignite',
 The vpc_manager is built from the Dockerfile located in services/vpc_manager folder and the container with the name vpm port 9001 is started.
 
 ## Preconditions
-Ensure that your target hosts, over which you plan to deploy the Busybox containers 
-1. have AlcorControl Agent running.
-2. No busy box containers with names like con1, con2 etc. are present (Delete them if they are).
+Ensure that your target hosts, over which you plan to deploy the Busybox containers
+1. Have Alcor Control Agent binaries located at /home/ubuntu/repos/aca/build/bin/AlcorControlAgent
 
 ## Running Test
-The main file for running the test is busybox_ping_test.py. It is a python script that can be run from command prompt in either of the following two ways:
-````
-python busybox_ping_test.py
+The main file for running the test is ping_test.py. It is a python script that can be run from command prompt in either of the following two ways:
+```
+python3 busybox_ping_test.py
 ./busybox_ping_test.py
-````
+```
 
-You can optionally provide the paramter --build to build all the docker images of the alcor services. This step need not be followed for any subsequent tests, unless any changes are made in Alcor.
+You can optionally provide the paramter "-b build" to build all the docker images of the alcor services. This step need not be followed for any subsequent tests, unless any changes are made in Alcor.
 	
 ## After Test Starts
 1. It will stop, remove existing Alcor services (if present) and start them all (as listed in alcor_services.ini file)
-2. Checks whether the Alcor Control Agents are running on the targets. If found not running, the test stops.
-3. Using the test info and payload, generate the end goal states for two end nodes.
-4. deploy two busy box containers con1 and con2 on the target hosts and performs the ping test.
+2. Checks the target hosts if any Alcor Control Agent (ACA) is running. If yes, it is killed and ACA restarted.
+3. Checks whether the ACAs are running on the targets. If found not running, the test stops.
+4. Using the test info and payload provided in config file, generate the end goal states for two end nodes.
+5. deploy two busy box containers con1 and con2 on the target hosts and runs a ping command from one container to another.
+
+## ACA on target hosts
+1. Following packages are required to build and run ACA. Though not mentioned in the commands, installing these pacakges will require sudo permissions.
+```
+openvswitch-switch
+openvswitch-common
+apache-pulsar-client
+apache-pulsar-client-dev
+```
+
```

Comment:
Where are these to be installed, target hosts?

Reply:
Yes, on target hosts where ACAs are supposed to run.

Reply:
Updated the README to add this point.